theme/components/button.scss

```diff
@@ -120,6 +120,18 @@ a.ck-button {
 		}
 	}
 
+	&:hover {
+		@include ck-tooltip_visible();
+	}
+
+	// Tooltip tweaks:
+	//   1. Get rid of the focus outline around the tooltip when focused (but not :hover).
+	//   2. Don't display tooltips for buttons with text; text explains the button anyway.
```

Comment:
It is similar case to disabled button, you can have disabled button with text, and explain in tooltip why is disabled. Or you can have button with label and show shortcut in tooltip.